isos/bootstrap/bootstrap

```diff
@@ -34,6 +46,27 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
 
     cp /bin/tether ${MOUNTPOINT}/.tether/tether
 
+
+    # linux-vdso.so.1 (0x00007ffc94d0d000)
```

Comment:
can you add a short note here explaining why we are doing this? Possibly with a "No Animals Were Harmed" disclaimer :)